scripts/k8s-storage/create-storageclasses.sh

```diff
@@ -25,3 +25,7 @@ do
 	sed "s/@@CLUSTER_ID@@/${FS_ID}/" "${sc}" |
 		kubectl create -f -
 done
+
+
+# Create default `csi` SubVolumeGroup
+kubectl -n rook-ceph exec "${TOOLBOX_POD}" -- ceph fs subvolumegroup create myfs csi
```

Comment:
@Madhu-1, external storage tests were failing because of subvolumegroup not present.
Now, added this to create default `csi` subvolumegroup.

Reply:
@iPraveenParihar i would suggest to create subvolumegroup CR with name `csi` using subvolumegroup CR here https://github.com/ceph/ceph-csi/blob/devel/scripts/rook.sh#L59